src/main/kotlin/com/exactpro/th2/codec/oracle/logminer/LogMinerTransformer.kt

```diff
@@ -138,15 +136,430 @@ class LogMinerTransformer(private val config: LogMinerConfiguration) : IPipeline
+            check(stage == Stage.BEGIN) {
+                "Incorrect stage for parsing insert into clause, expected: ${Stage.BEGIN}, actual: $stage, text: ${ctx.text}"
+            }
+            stage = Stage.PARSING_NAMES
+        }
+
+        override fun enterColumn_name(ctx: PlSqlParser.Column_nameContext) {
+            LOGGER.trace { "enterColumn_name token index: ${ctx.start.tokenIndex}, text: ${ctx.text}" }
+            check(stage == Stage.PARSING_NAMES) {
+                "Incorrect stage for parsing column name, expected: ${Stage.PARSING_NAMES}, actual: $stage, text: ${ctx.text}"
+            }
+
+            columNames.add(ctx.text.removeSurrounding("\""))
+        }
+
+        override fun exitInsert_into_clause(ctx: PlSqlParser.Insert_into_clauseContext) {
+            LOGGER.trace { "exitInsert_into_clause token index: ${ctx.start.tokenIndex}, text: ${ctx.text}" }
+            check(stage == Stage.PARSING_NAMES) {
+                "Incorrect stage for completing insert into clause, expected: ${Stage.PARSING_NAMES}, actual: $stage, text: ${ctx.text}"
+            }
+            stage = Stage.PARSED_NAMES
+        }
+
+        override fun enterValues_clause(ctx: PlSqlParser.Values_clauseContext) {
+            LOGGER.trace { "enterValues_clause token index: ${ctx.start.tokenIndex}, text: ${ctx.text}" }
+            check(stage == Stage.PARSED_NAMES) {
+                "Incorrect stage for parsing values clause, expected: ${Stage.PARSED_NAMES}, actual: $stage, text: ${ctx.text}"
+            }
+            stage = Stage.PARSING_VALUES
+        }
+
+        override fun enterUnary_expression(ctx: PlSqlParser.Unary_expressionContext) {
+            LOGGER.trace { "enterUnary_expression token index: ${ctx.start.tokenIndex}, text: ${ctx.text}" }
+            check(stage == Stage.PARSING_VALUES) {
+                "Incorrect stage for parsing values clause, expected: ${Stage.PARSING_VALUES}, actual: $stage, text: ${ctx.text}"
+            }
+            expressionHolder.add(UnaryExpression(ctx.start.tokenIndex))
+        }
+
+        override fun enterConstant(ctx: PlSqlParser.ConstantContext) {
+            LOGGER.trace { "enterConstant token index: ${ctx.start.tokenIndex}, text: ${ctx.text}" }
+            check(stage == Stage.PARSING_VALUES) {
+                "Incorrect stage for parsing values clause, expected: ${Stage.PARSING_VALUES}, actual: $stage, text: ${ctx.text}"
+            }
+
+            when {
+                ctx.NULL_() != null -> expressionHolder.last().apply(null)
+                ctx.numeric() != null -> {
+                    expressionHolder.last().apply(
+                        when {
+                            ctx.numeric().UNSIGNED_INTEGER() != null -> ctx.text.toLong()
+                            else -> ctx.text.toDouble()
+                        }
+                    )
+                }
+                ctx.quoted_string().isNotEmpty() -> { /* do nothing because this case is handled separate */ }
```

Comment:
Question: do we need to handle it separately? I don't see any specific logic in `enterQuoted_string` method. I am wondering if it is better to move this logic here and remote `enterQuoted_string` method completely

Reply:
quted_string token is used without constant wrapper too, for example in function argument